cms/bundles/utils.py

```diff
@@ -170,6 +170,31 @@ def get_preview_items_for_bundle(bundle: "Bundle", page_id: int, pages_in_bundle
     return preview_items
 
 
+def get_preview_teams_for_bundle(bundle: "Bundle") -> list[dict[Literal["id"], str]]:
+    """Get formatted preview teams for a bundle for API usage."""
+    team_identifiers = bundle.teams.values_list("team__identifier", flat=True)
+    return [{"id": identifier} for identifier in team_identifiers]
+
+
+def build_bundle_data_for_api(bundle: "Bundle") -> dict[str, Any]:
```

Comment:
Non-blocker: We don't expilicity unit test `build_bundle_data_for_api` and `get_preview_teams_for_bundle`. This bundles `utils.py` files has poor unit test coverage in general.

Can do as follow-up